[WIKI] Update minion-icons.md page

wontonstrips opened the pull request:
### Changes:

Uploaded minionicons.webp (picture from Magna's post on /sacrifice in discord)

The page is now complete with this picture showing the different minion icons that can be obtained and their respective sacrifice gp value needed to obtain them.

Updated November 26 2024:
Removed minionicons.webp
Added individual minion icons and adjust minion-icons.md

### Other checks:

- [x ] I have tested all my changes thoroughly.

gc commented:
You can get the individual images like this:

<:minion:660517408968146946><:amethystMinion:673561610228465664><:emeraldMinion:673561610366746635><:sapphireMinion:673561609834070047><:whiteMinion:673561609913892899><:yellowMinion:673561629115416616>

Get the number of the one you want and put it in this URL: https://cdn.discordapp.com/emojis/660517408968146946.png

And then we'd have to save each one to the images folder.

wontonstrips commented:
Removed minionicons.webp
Added individual minion icons and adjust minion-icons.md

"This branch is out-of-date with the base branch" - Do I need to "Update branch" or is that something you do?
If I need to do it should I choose "Update with merge commit" or "Update with rebase".
From what I saw online I am guessing rebase?  Sorry still learning and don't want to screw anything up. :)

gc commented:
No need if its just wiki changes.
